src/server/summary-log-upload/index.njk

```diff
@@ -67,30 +67,38 @@
         ]
       }) }}
 
-      {{ govukInsetText({
+        {{ govukInsetText({
         text: "You can submit an updated summary log whenever you need to add or adjust your waste records."
       }) }}
 
-      <form
+        <form
         class="form"
         action="{{ uploadUrl }}"
         method="post"
         enctype="multipart/form-data"
       >
-        {{ govukFileUpload({
-          id: "summary-log-upload",
-          name: "summaryLogUpload",
-          label: {
-            text: "Upload XLSX file"
-          },
-          javascript: true
-        }) }}
+          {% set summaryLogUploadError = null %}
+          {% if formErrors %}
+            {% set summaryLogUploadError = {
+              text: formErrors
+            } %}
+          {% endif %}
+          {{ govukFileUpload({
+            id: "summary-log-upload",
+            name: "summaryLogUpload",
+            label: { text: "Upload XLSX file" },
+            javascript: true,
+            attributes: {
+              accept: ".xlsx"
+            },
+            errorMessage: summaryLogUploadError
+          }) }}
 
-        {{ govukButton({
+          {{ govukButton({
           text: 'Continue'
         }) }}
-      </form>
+        </form>
+      </div>
     </div>
   </div>
-</div>
-{% endblock %}
+{% endblock %}
```

Comment:
The indentation looks to be inconsistent

Reply:
There definitely are indentation issues in the changed block from this, it was only a nitpick though